build_tools/cmake/iree_macros.cmake

```diff
@@ -252,3 +252,26 @@ function(iree_add_executable_dependencies EXECUTABLE DEPENDENCY)
     add_dependencies(${EXECUTABLE} ${DEPENDENCY})
   endif()
 endfunction()
+
+#-------------------------------------------------------------------------------
+# Tests
+#-------------------------------------------------------------------------------
+
+# iree_add_test_environment_properties
+#
+# Adds test environment variable properties based on the current build options.
+#
+# Parameters:
+# TEST_NAME: the test name, e.g. iree/base:ref_ptr_test
+# TARGET_NAME: the target name, e.g. iree_base_ref_ptr_test
```

Comment:
Seems this isn't used?

Reply:
Ah, leftover from when I also had
```
set_property(TEST ${_TEST_NAME} PROPERTY ENVIRONMENT "TEST_TMPDIR=${TARGET_NAME}_test_tmpdir")
```
here. `iree_lit_test` and `iree_cc_test` use slightly different paths (`"TEST_TMPDIR=${CMAKE_BINARY_DIR}/${_NAME}_test_tmpdir")` and Android doesn't specify a path at all, so I wasn't sure if unifying those made sense.